strax/processing/general.py

```diff
@@ -347,6 +401,13 @@ def _touching_windows(thing_start, thing_end,
 
     for i, t0 in enumerate(container_start):
         t1 = container_end[i]
+        if t0 < thing_start[right_i]:
+            # Container overlapped with previous one so we have to
+            # go back to first left_i before current container, and
+            # reset right_i
+            while left_i > 0 and thing_end[left_i] > t0 - window:
```

Comment:
I modified a bit of the `while` statement, from `left_i >= 0 and thing_end[left_i] >= t0 - window` to `left_i > 0 and thing_end[left_i] > t0 - window`.